### DISCUSSION TITLE
Identifying Observed Satellites that were not predicted

### DISCUSSION THREAD

**Discussion by JAY-Githb** [Stellarium/stellarium#3300] [Ideas]
With it's many enhancements, Stellarium could be very useful in identifying unknown satellites that are seen by chance.
Two requirements are a separate Config file and Satellites.json file.

There are a few ways to do this.
1. Run a 2d Stelarium and associated files in a different folder. I'm not sure how to do this.

2. Run one Stellarium normally with option to switch Sky and Viewing Options {F4), Config Window [F2], Stellites Config, and Satellites-2.json file. That would be similar to current ability to switch observing locations.

Predicting 25000 objects slows the program considerably so many features such as landscape, deep sky objects, cultures, etc. could be removed to speed up computing. 

Suggestions about the best way to accomplish this task are welcome.

**Comment by alex-w** [comment]
1. Please see the User Guide - it's possible with command line parameters for stellarium.exe file

**Comment by gzotti** [comment]
You can run several instances of Stellarium with different user data directories, see Guide chapter 6. Optionally they can be synchronized with the RemoteSync plugin, either completely, or with some properties excluded from synchronisation (to see something on one screen but not the other), see Guide ch.13.6.

The mere existence of landscapes, skycultures, DSO, ... does no harm. Just don't show DSO icons and images. It is really the existence and necessity to compute tens of thousands of satellites which is a problem, both for Stellarium and for the scientific community. Also having thousands of minor planets or comets slows down the program, so keep your data/ssystem_minor.ini small.

**Comment by JAY-Githb** [comment]
It took me a few hours over several days, but I figured out how to run 2 copies of Stellarium. 
I have since used it to identify some unknown satellites that crossed the field of view while I was looking for something else.

This is another very helpful function of Stellarium.
Thank you!

Jay
